ansible/smoke.yml

```diff
@@ -150,3 +150,28 @@
       become: false
       run_once: true
       when: smoke_enable_cdn_checks | default(True)
+
+- name: catalog-next.data.gov /dataset redirection
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: assert only one redirection to https and /dataset
+      vars:
+        url: http://catalog-next.data.gov
```

Comment:
According to the [bug](https://github.com/GSA/datagov-deploy/issues/2083), we should be testing **https** not http.

We don't really care if http://catalog.data.gov is two hops, as along as it's secure.

Why are are we testing catalog-next.data.gov and not catalog.data.gov?

Reply:
I fixed the URL, I used _http_ by mistake.
I also changed to test just the production URL.